Server/MirObjects/PlayerObject.cs

```diff
@@ -7684,7 +7684,7 @@ private void Rage(UserMagic magic)
 
             int duration = 18 + (6 * magic.Level);
             int maxDC = (int)Math.Round(Stats[Stat.MaxDC] * (0.12 + (0.03 * magic.Level)));
-            int minDC = (int)Math.Round(Stats[Stat.MinDC] * (0.12 + (0.03 * magic.Level)));
+            int minDC = (int)Math.Round(Stats[Stat.MaxDC] * (0.12 + (0.03 * magic.Level)));
```

Comment:
Why is minDC variable getting MaxDC stat?

Reply:
The skill ProtectionField's MinDC increase should be the same as MaxDC.
The code has been changed to a better way.